client/testserver/src/main/scala/sttp/tapir/client/tests/HttpServer.scala

```diff
@@ -133,8 +133,27 @@ class HttpServer(port: Port) {
           val e = q.enqueue
           WebSocketBuilder[IO].build(d, e)
         }
+
+    case r @ GET -> Root / "api" / "organization" =>
+      fromAcceptHeader(r) {
+        case "application/json" => organizationJson
+        case "application/xml"  => organizationXml
+      }
+
+    case r @ GET -> Root / "api" / "entity" =>
+      fromAcceptHeader(r) {
+        case "application/json" => personJson
+        case "application/xml"  => organizationXml
+      }
   }
 
+  private def fromAcceptHeader(r: Request[IO])(f: String => IO[Response[IO]]): IO[Response[IO]] =
```

Comment:
[minor] this could be a `PartialFunction` so that if a mapping is not defined, we get NotAcceptable as well

Reply:
this = `f`